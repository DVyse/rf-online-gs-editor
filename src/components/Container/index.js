/**
 *
 * Container
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import { Container as ContainerUI } from 'semantic-ui-react';

function Container({ children, ...props }) {
  return <ContainerStyled {...props}>{children}</ContainerStyled>;
}

Container.propTypes = {
  fluid: PropTypes.bool,
  children: PropTypes.node,
};

Container.defaultProps = {
  fluid: true,
  children: undefined,
};

export default Container;

const ContainerStyled = styled(ContainerUI)`
  padding: 0 15px;
`;
