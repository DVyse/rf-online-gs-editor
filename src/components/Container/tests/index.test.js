import React from 'react';
import { shallow } from 'enzyme';

import Container from '../index';

describe('<Container />', () => {
  it('Should exist', () => {
    const wrapper = shallow(<Container />);
    expect(wrapper).toMatchSnapshot();
  });
});
