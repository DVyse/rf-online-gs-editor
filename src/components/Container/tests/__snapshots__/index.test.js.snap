// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Should exist 1`] = `
<Container__ContainerStyled
  fluid={true}
/>
`;
